js/src/forum/components/FlagPostModal.js

```diff
@@ -46,6 +46,12 @@ export default class FlagPostModal extends Modal {
                 <input type="radio" name="reason" checked={this.reason() === 'off_topic'} value="off_topic" onclick={m.withAttr('value', this.reason)}/>
                 <strong>{app.translator.trans('flarum-flags.forum.flag_post.reason_off_topic_label')}</strong>
                 {app.translator.trans('flarum-flags.forum.flag_post.reason_off_topic_text')}
+                {this.reason() === 'off_topic' ? (
+                  <div>
+                    <p><em>{app.translator.trans('flarum-flags.forum.flag_post.optional_details')}</em></p>
```

Comment:
These should be marked up as a `<label>`. Styling should be achieved with CSS, not with `<em>` tags. (That said, I am not sold on the italic styling.)

When making this change, we should probably avoid nesting labels, too... :thinking: